[ci skip] Update guides on usage of --skip-solid flag to reflect latest information

jjatinggoyal opened the pull request:
<!--
Thanks for contributing to Rails!

Please do not make *Draft* pull requests, as they still send
notifications to everyone watching the Rails repo.

Create a pull request when it is ready for review and feedback
from the Rails team :).

If your pull request affects documentation or any non-code
changes, guidelines for those changes are [available
here](https://edgeguides.rubyonrails.org/contributing_to_ruby_on_rails.html#contributing-to-the-rails-documentation)

About this template

The following template aims to help contributors write a good description for their pull requests.
We'd like you to provide a description of the changes in your pull request (i.e. bugs fixed or features added), the motivation behind the changes, and complete the checklist below before opening a pull request.

Feel free to discard it if you need to (e.g. when you just fix a typo). -->

### Motivation / Background

<!--
Describe why this Pull Request needs to be merged. What bug have you fixed? What feature have you added? Why is it important?
If you are fixing a specific issue, include "Fixes #ISSUE" (replace with the issue number, remove the quotes) and the issue will be linked to this PR.
-->

The warning section on using `--skip-solid` misses mentioning Solid Cable, and exposes a non-existent command to enable solid-queue. This PR updates the warning section appropriately.

### Detail


### Additional information

<!-- Provide additional information such as benchmarks, references to other repositories, or alternative solutions. -->

### Checklist

Before submitting the PR make sure the following are checked:

* [x] This Pull Request is related to one change. Unrelated changes should be opened in separate PRs.
* [x] Commit message has a detailed description of what changed and why. If this PR fixes a related issue include it in the commit message. Ex: `[Fix #issue-number]`
* [x] Tests are added or updated if you fix a bug or add a feature.
* [x] CHANGELOG files are updated for the changed libraries if there is a behavior change or additional feature. Minor bug fixes and documentation changes should not be included.

Edouard-chin commented:
Thanks for the PR, that make sense to me.

Could you fix the linting issues ?

jjatinggoyal commented:
@Edouard-chin updated. Can you please merge the PR?

jjatinggoyal commented:
@Edouard-chin can you please review?

Edouard-chin commented:
I think the Github notifications are correctly working but thanks for checking.

---

Could you please fix the linting issue as mentioned in my previous comment? Also please squash your commits.

jjatinggoyal committed:
Update guides on usage of --skip-solid flag to reflect latest information

jjatinggoyal commented:
@Edouard-chin I'm sorry for multiple mentions. I've fixed the linting issue (all lines are less than 80 characters long) and squashed the commits now.

Edouard-chin commented:
Thanks, looks great!
